### ISSUE TITLE
build process complains about missing OpenGL header files

### ISSUE THREAD

**Issue by bdantas** [mpv-player/mpv-build#85]
I'm trying to build mpv on Ubuntu Trusty Tahr but I get an error about missing OpenGL header files for video output. 

I have installed every \*gl\*-dev package I could find, yet the build process continued to hang for the same reason. Please help me, as I'm totally stuck. How can I confirm whether I have the exact file(s) the build process is looking for? If I already have it/them as I suspect, how can I fix this issue?

**Comment by wiiaboo** [comment]
Upload build/config.log somewhere and paste the link, at the very least, or this will be closed for failing to follow the issue template.

**Comment by bdantas** [comment]
Here is build/config.log: 
https://pastebin.com/b8CBauNE

In case it helps, here all all installed packages with "gl" in package name (output of dpkg -l | grep gl):
https://pastebin.com/tGTbBKLu

**Comment by wiiaboo** [comment]
Did you check that mpv now needs a few X11 extensions https://github.com/mpv-player/mpv/commit/927afa311d4a89b022305508acbf4e7325fe3928 ?

**Comment by wiiaboo** [comment]
xinerama, xrandr and xscrnsaver are missing, more exactly.

**Comment by bdantas** [comment]
Yes, that did it. Build completed successfully after I installed all packages containing those three terms:
`sudo apt-get install libxinerama-dev x11-xserver-utils libxss-dev x11proto-xinerama-dev libxcb-xinerama0-dev libxrandr-dev` 
This might be more than necessary, but better to get more than not enough :)

The reason I got stuck is that a) those three extensions are not specifically mentioned in the Dependencies section of README.rst, and b) the error that is displayed when build fails only mentions OpenGL headers. Unless somebody is an mpv expert, this double whammy makes independent troubleshooting almost impossible.

Thank you very much for your help, wiiaboo. Much appreciated.
